footer.php

```diff
@@ -1,6 +1,15 @@
-
-<?php wp_footer() ?>
-
-</body> 
-
-</html>
\ No newline at end of file
+<link rel="stylesheet" href="/assets/css/ywca.css">
```

Comment:
Remove stylesheet link here and fix functions.php enqueue scripts, that should import ywca.css